docs/Plugins.rst

```diff
@@ -1109,6 +1109,27 @@ Example:
     This will confiscate rotten and dropped food, garbage on the floors and any
     worn items with 'X' damage and above.
 
+.. _dwarfvet:
```

Comment:
I think this file is sorted alphabetically, so this should be after dwarfmonitor.